tasks/StreamerTask.cpp

```diff
@@ -398,19 +398,23 @@ void jsonRead(T& field, Json::Value& json, string const& fieldName) {
     field = jsonConvert<T>(json[fieldName]);
 }
 
-InboundStreamStatistics parseInboundStreamStatistics(Json::Value data) {
+InboundStreamStatistics parseInboundStreamStatistics(Json::Value data)
+{
     InboundStreamStatistics ret;
```

Comment:
Who consumes `InboundStreamStatistics` in the end, dashboard?

Reply:
Nobody. It's being logged for post-mortem analysis.

The "dashboard" (that is, what we call the HUD) is the one sending that information.